app/worker.php

```diff
@@ -33,28 +35,25 @@
 use Utopia\Logger\Logger;
 use Utopia\Platform\Service;
 use Utopia\Pools\Group;
+use Utopia\Queue\Broker\Pool as BrokerPool;
 use Utopia\Queue\Message;
 use Utopia\Queue\Publisher;
 use Utopia\Queue\Server;
 use Utopia\Registry\Registry;
 use Utopia\System\System;
 
 Authorization::disable();
-Runtime::enableCoroutine(SWOOLE_HOOK_ALL);
```

Comment:
why did we remove this? AFAIK this is required to enable the swoole curl extension which is required for making async requests in our workers

Reply:
It's the parameter default so it's functionally the same